storybook/.storybook/preview.js

```diff
@@ -9,10 +9,10 @@ import i18n from "utils/i18n"; // importing the same i18n
 
 import "assets/stylesheets/App.scss";
 import "assets/stylesheets/index.scss";
-
+import "assets/stylesheets/App.scss";
 // .storybook/preview.jsx
 import { I18nextProvider } from "react-i18next";
-
+import "assets/stylesheets/App.scss";
```

Comment:
We have this import twice, one can be removed?

Reply:
I don't think it's needed at all, let alone twice - have deleted both!